### PR TITLE
#103 Docs Update UI Overview v1.3.0

### PR DESCRIPTION
**Pull request by josevalverdeinbibo**
### Tasks

- Added new screenshots for AdnFat in "UI Overview" page.
- Added content for AdnFat in "UI Overview" page.
- Updated shelf and/or menu for all screenshots in "A Simple Setup" page.
- Updated screenshot (figure 9) in "Skin" page.
- Updated screenshot (figure 1) in "Simshape" page.

Issue of this PR: #103

### CODE REVIEW

**Comment by carlosmonteagudoinbibo** [comment]
Can you please fix the separators in this table? They are not coherent with the separators on the shelf:

image

File: docs/maya/ui_overview.md [hunk 1]
```diff
@@ -114,6 +115,7 @@ Both in Locators and Sensors there are option boxes to launch a simple UI to pro
 - **Skin**. Creates an AdnSkin deformer to the selected mesh. This item corresponds to the shelf button ![Create AdnSkin](images/adn_skin.png){style="width:4%"}.
 - **Ribbon Muscle**. Creates AdnRibbonMuscle deformer to the selected mesh. The deformer will be applied to the last element in the selection. Other elements in the list (optional) will be considered as attachment objects. This item corresponds to the shelf button ![Create AdnRibbonMuscle](images/adn_ribbon_muscle.png){style="width:4%"}.
 - **Muscle**. Creates AdnMuscle deformer to the selected mesh. The deformer will be applied to the last element in the selection. Other elements in the list (optional) will be considered as attachment objects. This item corresponds to the shelf button ![Create AdnMuscle](images/adn_muscle.png){style="width:4%"}.
+- **Fat**. Applied an AdnFat deformer to the selected mesh. The deformer will be applied to the second element in the selection. The first element will be considered as base mesh. This item corresponds to the shelf button ![Create AdnFat](images/adn_fat.png){style="width:4%"}.
```
Comment (carlosmonteagudoinbibo) [inline docs/maya/ui_overview.md:118]: Please, move fat right after skin. We are going to apply the same change in the menu. The screenshot will need to be updated

Comment (carlosmonteagudoinbibo) [inline docs/maya/ui_overview.md:118]:
```suggestion
- **Fat**. Applies an AdnFat deformer to the selected mesh. The deformer will be applied to the second element in the selection. The first element will be considered as base mesh. This item corresponds to the shelf button ![Create AdnFat](images/adn_fat.png){style="width:4%"}.
```

**Review by carlosmonteagudoinbibo — CHANGES_REQUESTED** [review]

File: docs/maya/ui_overview.md (hunk 1 above)
Comment (josevalverdeinbibo) [reply docs/maya/ui_overview.md:118]: Fixed.

Comment (josevalverdeinbibo) [reply docs/maya/ui_overview.md:118]: Done.

**Review by carlosmonteagudoinbibo — APPROVED** [review]

File: docs/maya/images/adonis_menu_solvers.png
Comment (lucasferryinbibo) [inline docs/maya/images/adonis_menu_solvers.png:1]: Reminder to update this file

**Review by lucasferryinbibo — APPROVED** [review]

**Pull request merged**